.travis.yml

```diff
@@ -4,7 +4,7 @@ python:
   - "2.7"
 
 install:
-    - pip install -r requirements.txt
+    - pipenv install
```

Comment:
This installs everything in the a virtual env (in /home/user/.local/share/...). When running in travis, I think we can use a system installation instead (see pipenv install -h).

Using pipenv, all the requirements are installed in a virtual evn, so we must run both flake8 and pytest inside the virtual env.

We need to check if pipenv install --system create a virtual env, and use the virtual env (e.g. pipevn run flake8) to run the tests.